client/src/components/survey-panel/survey-panel.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import Draggable from 'react-draggable'
+import InlineSVG from 'svg-inline-react'
+import classNames from 'classnames/bind'
+import styles from './survey-panel.scss'
+import CloseIcon from '../../assets/svg/baseline-close-24px.svg'
+
+const css = classNames.bind(styles)
+
+// eslint-disable-next-line react/prefer-stateless-function
+class SurveyPanel extends React.Component {
+  render() {
+    const { hidden, onClick } = this.props
+
+    return (
+      <div className={hidden ? css('panel-container', 'hidden') : css('panel-container')}>
```

Comment:
`className={css('panel-container', { hidden })}` works here

Reply:
Hmm, so `css({a: false, b: true})` resolves to `class=b`?

Reply:
yes